packages/base/catalog-entry.gts

```diff
@@ -102,31 +71,116 @@ export class CatalogEntry extends CardDef {
   };
 
   static isolated = class Isolated extends Component<typeof this> {
+    get icon() {
+      return this.args.model.constructor?.icon;
+    }
+
     <template>
-      <CatalogEntryContainer class='container'>
```

Comment:
Is `CatalogEntryContainer` used anymore? Should it be removed as part of this PR?

Reply:
This is used inside of the fitted view. We can just make this local to the fitted view if preferred since its not being re-used any where else

Reply:
I don't have a strong preference. Just wanted to make sure we didn't have an orphan.